api_poll/api-server-poll.yml

```diff
@@ -30,8 +30,23 @@
 
             if [ $COUNTER -lt 48 ]; then
                 if [ ! -f do_not_poll ]; then
+
                   sudo docker tag registry.access.redhat.com/rhel7:latest {image_under_test}
-                  STATUS_CODE=$(sudo SERVER={analytics_server} IMAGE_NAME={image_under_test} GITURL={git_url} GITSHA={git_sha} atomic scan --verbose --scanner=analytics-integration --scan_type=report {image_under_test}|grep api_status_code|cut -f2 -d ":"|cut -c2-4)
+
+                  echo "Running the analytics-scanner report scan_type and finding the status_code returned from API server.."
+
+                  sudo SERVER={analytics_server} IMAGE_NAME={image_under_test} GITURL={git_url} GITSHA={git_sha} atomic scan --verbose --scanner=analytics-integration --scan_type=report {image_under_test} > scanner_response.txt
+
+                  cat scanner_response.txt
+
+                  STATUS_CODE=$(cat scanner_response.txt |grep api_status_code | cut -f2 -d ":" | cut -c2-4)
+
+                  OUTPUT_FILE=$(cat scanner_response.txt | grep "Files associated with this scan are in" | cut -d ' ' -f 8 | sed 's/\.//')
+
+                  echo "Removing the unneeded scanner results file $OUTPUT_FILE"
+
+                  sudo rm -rf $OUTPUT_FILE scanner_response.txt
```

Comment:
OUTPUT_FILE here is a variable not file right? why we are removing it?

Reply:
I think that's the whole point! 
$OUTPUT_FILE is a variable which has value=result file for the scanner which is no longer used but is now exported on the local file system, piling up with every polling job.

The aim of the PR is to remove that file after we read the status code.

@bamachrn